module/cli_controller.py

```diff
@@ -55,11 +56,15 @@ def make_path_string(path_list) -> str:
         """
         return ' / '.join(path_list) + ":"
 
+    def set_target(self, target) -> None:
+        self.target = target
+
     def print_help(self, node) -> None:
         """
         Prints the info and any children or parameters of the node.
         """
         print('( ' + node.name + ' )')
+        print("Currently selected robot: {}\n".format(self.target))
```

Comment:
should this be shown every time one asks for help on a node?

Reply:
I think yes. Why not? It is probably at least as important as knowing what node you executed help in. If you run a command on the wrong robot you might send into water or holes.

Reply:
As it is not really valuable information if you want to just know what a particular command does, I'd say there should be a separate command to get to know the currently selected robot/swarm.

Reply:
any intuitive suggestions?

Reply:
And remember that if people don't know what to do they will type "help". And the user should then have all the tools he needs to continue operating the robot.

Reply:
Maybe the `select` command without any parameters should show the currently selected? Or something along the lines of that.

Reply:
how do you know selecting a robot is even a thing then?